src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx

```diff
@@ -25,7 +25,7 @@ const HorizontalLayout = styled.View`
   flex-direction: row;
 `
 
-const Card = styled(VerticalLayout)`
+const Card = styled.View`
```

Comment:
The problem with the row collapsing is `VerticalLayout` includes `flex: 1` which messes with the height of the rows

Reply:
👍